lib/forklift/box_distributor.rb

```diff
@@ -187,6 +187,20 @@ def configure_libvirt(machine, box, networks = [])
         p.memory = box.fetch('memory').to_i * @settings['scale_memory'].to_i if box.fetch('memory', false)
         p.machine_virtual_size = box.fetch('disk_size') if box.fetch('disk_size', false)
 
+        if box.fetch('add_volumes', false)
```

Comment:
I think this can be slightly simplified to ```box.fetch('add_volumes', []).each do |volume|```

Reply:
Thanks for the hint.